src/unittests/f32_interval.spec.ts

```diff
@@ -2691,6 +2692,44 @@ interface PointToVectorCase {
       );
     });
 
+  g.test('unpack2x16floatInterval')
+    .paramsSubcasesOnly<PointToVectorCase>(
+      // prettier-ignore
+      [
+        // Some of these are hard coded, since the error intervals are difficult to express in a closed human readable
+        // form due to the inherited nature of the errors.
+        // f16 normals
+        { input: 0x00000000, expected: [[0], [0]] },
+        { input: 0x80000000, expected: [[0], [0]] },
+        { input: 0x00008000, expected: [[0], [0]] },
+        { input: 0x80008000, expected: [[0], [0]] },
+        { input: 0x00003c00, expected: [[1], [0]] },
+        { input: 0x3c000000, expected: [[0], [1]] },
+        { input: 0x3c003c00, expected: [[1], [1]] },
+        { input: 0xbc00bc00, expected: [[-1], [-1]] },
+        { input: 0x49004900, expected: [[10], [10]] },
+        { input: 0xc900c900, expected: [[-10], [-10]] },
+
+        // f16 subnormals
+        { input: 0x000003ff, expected: [[0, kValue.f16.subnormal.positive.max], [0]] },
+        { input: 0x000083ff, expected: [[kValue.f16.subnormal.negative.min, 0], [0]] },
+
+        // f16 out of bounds
+        { input: 0x7c000000, expected: [kAny, kAny] },
```

Comment:
Only one of the lanes here is OOB. Do we expect that to affect the other lane?

Reply:
Theoretically it shouldn't, but practically we have seen on the packing side drivers that mangle the non-OOB lane.

Reply:
Gross. Okay then!

Reply:
There is probably a very depressing book to be written on all of the little gross things that happen in the graphics stack that make me sad :upside_down_face: